app/src/main/java/fr/free/nrw/commons/edit/EditActivity.kt

```diff
@@ -239,6 +245,12 @@ class EditActivity : AppCompatActivity() {
      * as a result, and finishes the current activity.
      */
     fun getRotatedImage() {
+        //Get Permission to saccess
+        if (ContextCompat.checkSelfPermission(this, Manifest.permission.WRITE_EXTERNAL_STORAGE)
+            != PackageManager.PERMISSION_GRANTED) {
+            ActivityCompat.requestPermissions(this, arrayOf(Manifest.permission.WRITE_EXTERNAL_STORAGE), 7747)
```

Comment:
Would you mind adding a comment explaining what 7747 is, or maybe use constants rather than integer?

Reply:
Ok ! Actually it's just a randomly choice code, cause it should be avoided to be repeat

Reply:
If I understand the documentation correctly, [onRequestPermissionsResult](https://developer.android.com/reference/androidx/core/app/ActivityCompat.OnRequestPermissionsResultCallback#onRequestPermissionsResult(int,java.lang.String[],int[])) has to use the same integer? Do we have a onRequestPermissionsResult method somewhere that should use this code?

Reply:
Currently not, but I'm not sure if anyone will use it in the future.
This code(Integer) is a necessary parameter for requestPermissions to run.
Since there have been no instances of request failures in my testing environment so far, there isn't a situation that handle request failures about it in onRequestPermissionsResult (It always support to work)
I am not sure what the program should do in the event of a request failure. If only add something like this:
Toast.makeText(context, "Fail to request permission for access picture", Toast.LENGTH_SHORT).show()
When it fail is fine, then I can add it.

Reply:
No need for a failure check/toast.

Would you mind making the integer final static as seen at https://github.com/raulhaag/MiMangaNu/blob/master/app%2Fsrc%2Fmain%2Fjava%2Far%2Frulosoft%2Fmimanganu%2FMainActivity.java#L48 ?

Reply:
Okay, I will do it this way!